documentation/book/appendix_metrics.adoc

```diff
@@ -37,66 +37,160 @@ endif::Kubernetes[]
+[source,shell,subs="+quotes"]
+oc apply -f prometheus-operator-service-account.yaml
+oc apply -f prometheus-operator-cluster-role.yaml
+oc apply -f prometheus-operator-cluster-role-binding.yaml
+oc apply -f prometheus-operator-deployment.yaml
+
+The {ProductName} repository contains configuration files for the Prometheus server.
+When you apply these configuration files, the following resources are created in your {OpenShiftName} cluster and managed by the Prometheus Operator.
+
+* A `ClusterRole` that grants permissions to read the Prometheus health endpoints of the {OpenShiftName} system, including cAdvisor and the kubelet for container metrics. The Prometheus server configuration uses the {OpenShiftName} service discovery feature in order to discover the pods in the cluster from which it gets metrics.  For this feature to work correctly, the service account used for running the Prometheus service pod must have access to the API server so it can retrieve the pod list.
 * A `ServiceAccount` for the Prometheus pods to run under.
 * A `ClusterRoleBinding` which binds the aforementioned `ClusterRole` to the `ServiceAccount`.
-* A `Deployment` to manage the actual Prometheus server pod.
-* A `ConfigMap` to manage the configuration of Prometheus Server.
-* A `Service` to provide an easy to reference hostname for other services to connect to Prometheus server (such as Grafana).
+* A `Deployment` to manage the Prometheus Operator pod.
+* A `ServiceMonitor` to manage the configuration of the Prometheus pod.
+* A `Secret` to manage additional Prometheus settings.
+* A `PrometheusRule` to manage alert rules for the Prometheus pod.
+* A `Prometheus` to manage the configuration of the Prometheus pod.
 
-Prometheus also provides an alerting system through the link:https://prometheus.io/docs/alerting/alertmanager/[alert manager] component.
-In order to enable alerting, the provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml[`alerting-rules.yaml`] file describes a `ConfigMap` resource which defines sample alerting rules on Kafka and Zookeeper metrics.
-When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the alert manager which then uses the configured notification methods to notify the user.
+==== Deploying the Prometheus Operator on {KubernetesName}
 
-More information about how to set up alerting rules https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[here].
+To deploy the Prometheus Operator in your Kafka cluster, apply the `.yaml` files from the https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository].
 
-==== Deploying on {OpenShiftName}
+To use a different namespace than that specified in the repository files, use the following commands to download and edit the files from the repository:
```

Comment:
Same as above, please mention which namespace does it use to make it easier for users to decide if this is different namespace or not.